plot-api/src/main/kotlin/jetbrains/letsPlot/scale/XYContinuous.kt

```diff
@@ -31,7 +31,7 @@ fun scale_x_continuous(
     breaks: List<Number>? = null,
     labels: List<String>? = null,
     limits: Pair<Number?, Number?>? = null,
-    expand: Any? = null,
+    expand: List<Number>? = null,
```

Comment:
Let's change to `List<Any>` and add size and elements type checking similar to `scale_color_grey`

Reply:
Type was changed and checks were added